test/bin/pyutils/build_bootc_images.py

```diff
@@ -346,6 +346,100 @@ def process_image_bootc(groupdir, bootcfile, dry_run):
         os.rename(f"{bf_outdir}/bootiso/install.iso", bf_targetiso)
 
 
+def process_container_encapsulate(groupdir, containerfile, dry_run):
+    ce_path = os.path.join(groupdir, containerfile)
+    ce_outname = os.path.splitext(containerfile)[0]
+    ce_outdir = os.path.join(BOOTC_IMAGE_DIR, ce_outname)
+    ce_logfile = os.path.join(BOOTC_IMAGE_DIR, f"{ce_outname}.log")
+    ce_targetimg = os.path.join(ce_outdir, "manifest.json")
+    ce_tagname = "build_image_tag"
+    ce_tagval = f"localhost/{ce_outname}:latest"
+
+    def get_images_by_label():
+        images_args = [
+            "sudo", "podman", "images",
+            "--filter", f"label={ce_tagname}={ce_tagval}",
```

Comment:
What is advantage of using labels in this process instead of using tags directly? Is it producing more than 1 image?

Reply:
When running `rpm-ostree compose container-encapsulate` command, we need to somehow `mark` the produced image and it seems to only provides this functionality using `--label` option.
Once the image is produced, subsequent commands find it by the label.

Reply:
Gotcha, we cannot `podman load` with a `--tag` - without the labels we wouldn't find what we just loaded.